experimenter/tests/integration/tox.ini

```diff
@@ -22,7 +22,7 @@ commands =
     pip install -r ../requirements.txt
     pytest --verify-base-url --base-url https://nginx/nimbus/ --html=test-reports/report.htm --self-contained-html --driver Firefox nimbus/test_mobile_targeting.py {posargs} -vvv
 setenv =
-    PYTHONPATH=/application-services/components/nimbus/src
```

Comment:
I suspect this might be the problem. Does the new path point to where the nimbus.py file is created?

Reply:
I thought the same thing, but it should be the correct place: 
```COPY --from=rust-builder /application-services/megazord/nimbus.py /application-services/megazord/rust_sdk.py```